Nodify/Themes/Styles/Node.xaml

```diff
@@ -75,6 +75,7 @@
 
                                 <!--Data IN-->
                                 <ItemsControl ItemsSource="{TemplateBinding Input}"
+                                              VerticalAlignment="{TemplateBinding VerticalAlignment}"
```

Comment:
Make sure you set the default `VerticalAlignment` to `Top` to avoid breaking changes.